Proxy/proxy.sample.ini

```diff
@@ -6,9 +6,16 @@ PORT=8000
 FILENAME=log.db
 SCHEMANAME=db.sql
 
+[TESTDATABASE]
+FILENAME=log.test.db
+
 [PROXY]
 LOG=0
 UNITS=1
```

Comment:
Just realized if we can have more than one unit in the test mode then we need to be able to identify which unit is which callsign-nodeid. We did this by using separate sections as shown elsewhere in this INI file. However that could start to get messy... Would something like `TESTCALLSIGN0` and similar be acceptable here?